README.md

```diff
@@ -62,6 +62,11 @@ Tool                  | Description
 [Kaminari]            | Elegant pagination for any sort of collection
 [Ransack]             | Provides a simple search API to query your data
 
+## Security contact information
+
+To report a security vulnerability, please use the [Tidelift security contact].
```

Comment:
Just a small update to make the link the action:

```suggestion
Please use the Tidelift security contact to [report a security vulnerability][Tidelift security contact].
```

Reply:
Nice, thanks! I'm committing the suggestion now, and then let's squash it with the other commit when merging.

Reply:
Ok great! I'll get it out now. Thanks!